site/docs/Tutorial-Mutations.md

```diff
@@ -87,16 +87,16 @@ var fakeDatabase = {};
 
 // The root provides the top-level API endpoints
 var root = {
-  getMessage: function({author}) {
+  getMessage: function ({author}) {
```

Comment:
In particular in https://github.com/lacker/graphql.github.io/blob/2c1bb0d86257d7fc66c86d5a4a85064b7717beaf/site/docs/Tutorial-GettingStarted.md we say, hey you might want to read up on es6 classes, promises, and fat arrow functions. I think those are the only es6 features in all this, plus the fact that backticks let you do multiline strings, which is kind of obvious what it does from example code.